concepts/team-changenotification-channels.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Get change notifications for change in creation, updation or deletion of channels using Microsoft Graph"
+description: "Get change notifications for change in creation, updation or deletion of channels using Microsoft Graph"
+author: "AbhishekAnand"
+localization_priority: Priority
+ms.prod: "microsoft-teams"
+ms.custom: scenarios:getting-started
+---
+
+# Get change notifications for channel using Microsoft Graph
+
+Change notifications enable you to subscribe to changes (create, update, and delete) to a [channel](/graph/api/resources/channel?preserve-view=true). Change notifications provide a low latency model by allowing you to maintain a [subscription](/graph/api/resources/webhooks?preserve-view=true). You can also get the resource data in the notifications and therefore avoid calling the API to get the payload.
```

Comment:
'changes to a channel' sounds like we support notifications for individual channels too, which is not true. We only support notifications for all channels inside a given team, but not at an individual channel level.
We support /teams/id/channels, not /teams/id/channels/id